querybook/server/lib/ai_assistant/ai_assistant.py

```diff
@@ -10,6 +11,10 @@ def __init__(self, provider: str, config: dict = {}):
         self._assisant = get_ai_assistant_class(provider)
         self._assisant.set_config(config)
 
+    @property
+    def current_user_id(self):
```

Comment:
this ties AIAssistant to only work within Flask API, what if we want to invoke it not in web server? Can you make generate_title_from_query take in user id?